hooks/command

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+export AWS_PAGER=""
+
+build="${BUILDKITE_BUILD_NUMBER:-0}"
```

Comment:
Do we want to throw if this is not available, rather than defaulting to `0`?

Reply:
Yep 👍